crypto/bn/bn_mod.c

```diff
@@ -110,6 +111,61 @@ int BN_mod_sub(BIGNUM *r, const BIGNUM *a, const BIGNUM *b, const BIGNUM *m,
     return BN_nnmod(r, r, m, ctx);
 }
 
+/*
+ * BN_mod_sub variant that may be used if both a and b are non-negative,
+ * a is less than m, while b is of same width as m. It's implemented as
```

Comment:
Nit: Perhaps s/width/bit width/, so be clear you're not talking about word width.